.github/workflows/ci.yml

```diff
@@ -125,35 +137,18 @@ jobs:
           # setup docker containers
           docker-compose up -d --build
 
-      - name: Run cypress
-        run: |
-          cd projects/plio-frontend/
-
-          # delete the node_modules created by the docker
-          echo 'deleting node modules'
-          rm -rf node_modules/
-
-          # install dependencies from the current shell user
-          echo 'running npm'
-          npm install
-          echo 'finished npm install'
-
-          # setup env secrets
-          export cypress_backend_convert_social_auth_token_url=${{ secrets.CYPRESS_PLIO_BACKEND_CONVERT_SOCIAL_AUTH_TOKEN_URL }}
-          export cypress_backend_client_id=${{ secrets.DJANGO_DEFAULT_OAUTH2_CLIENT_ID }}
-          export cypress_backend_client_secret=${{ secrets.DJANGO_DEFAULT_OAUTH2_CLIENT_SECRET }}
-          export cypress_auth_google_refresh_token=${{ secrets.CYPRESS_AUTH_GOOGLE_REFRESH_TOKEN }}
-          export cypress_auth_google_client_id=${{ secrets.GOOGLE_OAUTH2_CLIENT_ID }}
-          export cypress_auth_google_client_secret=${{ secrets.GOOGLE_OAUTH2_CLIENT_SECRET }}
-
-          # run cypress test cases
-          npx cypress run --record --key ${{ secrets.CYPRESS_RECORD_KEY }}
-
-      - name: Coveralls
-        uses: coverallsapp/github-action@master
+      - name: Install TestCafe BrowserStack plugin
+        run: npm install testcafe-browser-provider-browserstack
+
+      - name: Run Windows 10 tests
+        uses: DevExpress/testcafe-action@latest
+        with:
+          args: "'browserstack:chrome@96.0:Windows 10','browserstack:firefox@95.0:Windows 10','browserstack:edge@96.0:Windows 10' projects/plio-frontend/tests/integration --skip-js-errors --test-scheduling"
```

Comment:
This is available in BrowserStack documentation.
https://www.browserstack.com/docs/automate/selenium/getting-started/nodejs/testcafe/run-tests-in-parallel

Since we're not using the testcafe version forked by BrowserStack, I'm removing this additional parameter.